CMakeLists.txt

```diff
@@ -269,6 +269,7 @@ endif (CMAKE_C_SIZEOF_DATA_PTR EQUAL 8)
 if (APPLE AND X64)
   # XXX #1979: 64-bit OSX is not supported.
   message(WARNING "64-bit Mac OSX is not supported")
+  set(MACOS64 1)
```

Comment:
Ok. I added this for convenience since I was adding a lot. I agree, we can do w/o if not completely necessary.